### PR TITLE
fix embedding and pile-up 

### PR DESCRIPTION
**Pull request by AllaMaevskaya**
now embedding and pile-up works

### CODE REVIEW

File: Detectors/FIT/FT0/simulation/src/Digitizer.cxx [hunk 1]
```diff
@@ -52,6 +52,8 @@ double Digitizer::get_time(const std::vector<double>& times)
     is_positive = cfd_val > 0;
   }
   LOG(INFO) << "CFD failed to find peak ";
+  for (double t : times)
```
Comment (sawenzel) [inline Detectors/FIT/FT0/simulation/src/Digitizer.cxx]: do you need these printouts?

File: Detectors/FIT/FT0/simulation/src/Digitizer.cxx [hunk 2]
```diff
@@ -88,19 +90,15 @@ void Digitizer::process(const std::vector<o2::ft0::HitType>* hits)
     Double_t hit_time = hit.GetTime() - time_compensate;
 
     //  bool is_hit_in_signal_gate = (abs(hit_time) < parameters.mSignalWidth * .5);
-
+    if (hit_time < -12.5 || hit_time > 12.5)
```
Comment (sawenzel) [inline Detectors/FIT/FT0/simulation/src/Digitizer.cxx]: could you please use meaningful constant names for these hard-coded values?

File: Steer/DigitizerWorkflow/src/FT0DigitizerSpec.cxx [hunk 3]
```diff
@@ -119,20 +120,19 @@ class FT0DPLDigitizerTask
...
-        labels.clear();
         mDigitizer.setEventID(collID);
         mDigitizer.setSrcID(part.sourceID);
         mDigitizer.process(&hits);
-        // copy labels into accumulator
-        labelAccum.mergeAtBack(labels);
       }
       mDigitizer.setDigits(mDigitsBC, mDigitsCh);
+      // copy labels into accumulator
+      //   labelAccum.mergeAtBack(labels);
```
Comment (sawenzel) [inline Steer/DigitizerWorkflow/src/FT0DigitizerSpec.cxx]: this looks weird. So does it mean that the digitizer itself accumulates labels for all events? There is no flushing in between? Probably the same is done for digits?

**Comment by sawenzel** [comment]
Thanks for this effort. I'll take a deeper look and test it in action.

**Comment by sawenzel** [comment]
Dear Alla. I confirm that it now passes the test in the scenario with a very high interaction rate.

However, if I choose a low interaction rate (see script $O2_ROOT/prodtests/check_embedding_pileup.sh), I still see the flag "1" for pileup which I believe shouldn't be the case ... unless you construct a single digit object for the whole timeframe.

File: Detectors/FIT/FT0/simulation/src/Digitizer.cxx (hunk 1 above)
Comment (AllaMaevskaya) [reply Detectors/FIT/FT0/simulation/src/Digitizer.cxx]: yes, I want to know this

**Comment by sawenzel** [comment]
Dear Alla. Did you see my last comment? Please let me know when it works also with very low interaction rate.

**Comment by AllaMaevskaya** [comment]
I constrain 1 digits object per simulated event, and in ft0digits.root I have 4 identical events with the same interaction record.

**Comment by AllaMaevskaya** [comment]
OK, we will discuss with electronics people more detail digitization procedure and solution will come with its implementation

**Comment by AllaMaevskaya** [comment]
Thank you. Now i understand.

**Comment by AllaMaevskaya** [comment]
Dear Sandro and Ruben,

i think now should be OK

**Comment by sawenzel** [comment]
Thank you Alla! I will test it separately before merging.

**Comment by sawenzel** [comment]
The test looks good.

File: DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/Digit.h [hunk 4]
```diff
@@ -19,9 +19,12 @@
 #include "CommonDataFormat/RangeReference.h"
 #include "CommonDataFormat/TimeStamp.h"
 #include "DataFormatsFT0/ChannelData.h"
+#include <FairLogger.h>
 #include <Rtypes.h>
 #include <gsl/span>
 #include <bitset>
+#include <iostream>
+#include "FairLogger.h"
```
Comment (sawenzel) [inline DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/Digit.h]: do you need iostream/FairLogger in the header? Better to avoid.

Comment (AllaMaevskaya) [reply DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/Digit.h]: removed now

**Pull request merged**